custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -29,6 +29,17 @@
 from custom_components.elasticsearch.system_info import SystemInfo
 
 ALLOWED_ATTRIBUTE_TYPES = tuple | dict | set | list | int | float | bool | str | None
+SKIP_ATTRIBUTES = [
+    "friendly_name",
+    "id",
+    "type",
+    "entity_picture",
+    "icon",
+    "entity_id",
```

Comment:
I also haven't seen this attribute, but I'm not opposed to excluding this if you're finding it pervasive and unhelpful.

Reply:
In almost all cases it just contains the entity id but for `group` sensors and `binary` sensors it appears to be an array of the child entity id's so I will probably re-add it